cli.js

```diff
@@ -8,6 +8,7 @@ const psList = require('ps-list');
 const numSort = require('num-sort');
 const escExit = require('esc-exit');
 const cliTruncate = require('cli-truncate');
+const portToProc = require('portproc').portToProc;
```

Comment:
Would have been more readable as:

```js
const portToProcess = require('portproc').portToProcess;
```

Not really worth saving a few characters for limited gain.